src/main/java/Hw/Main.java

```diff
@@ -2,6 +2,56 @@
 
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("Домашнее задание № 9");
+        System.out.println("Задание 1");
+        int[] payOfMonday = {25000, 30000, 28000, 35000, 33000};
+        int sumOfMonth = 0;
+        for (int i = 0; i < payOfMonday.length; i++) {
+            sumOfMonth = sumOfMonth + payOfMonday[i];
+        }
+        System.out.println("Сумма трат за месяц составила " + sumOfMonth + "  рублей");
+        System.out.println("Задание 2");
+        int[] payOfDay = {35000, 30000, 29000, 31000, 33000};
+        int minOfPay = payOfDay[0];
+        int maxOfPay = payOfDay[0];
+        for (int i = 1; i < 5; i++) {
+            final int current=payOfDay[i] ;
+            if (current > maxOfPay) {
+                maxOfPay = current;
+            }
+            if (current < minOfPay) {
+                minOfPay = current;
+            }
+        }
+        System.out.println("Минимальная сумма трат за неделю составила " + minOfPay + " рублей");
+        System.out.println("Максимальная сумма трат за неделю составила " + maxOfPay + " рублей");
+        //Задание 3 begin
+        System.out.println("Задание 3 ");
+        int[] payOfWeek = new int[5];
```

Comment:
можно было не создавать новую переменную, а просто переиспользовать ранее объявленную
```suggestion
        payOfMonday = new int[5];
```